### PR TITLE
Cbridge Update: Sentinel unrelaxed

### PR DESCRIPTION
**Pull request by sekuba**
Closes L2B-5000

### CODE REVIEW

File: packages/backend/discovery/cbridge/ethereum/diffHistory.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+```diff
+    contract Sentinel (0xF140024969F6c76494a78518D9a99c8776B55f70) {
+    +++ description: The Sentinel is itself a Governor and Pauser in the bridge contracts. It allows additional Sentinel-Governors to make changes in its name that can be restricted by Guards changing the Sentinel's relaxed state.
++++ description: Number of relaxed guards in the Sentinel.
+      values.numRelaxedGuards:
+-        2
++        0
++++ description: An unrelaxed Sentinel allows only parameter changes in the Bridge that would make it more secure (decrease limits, increase delay period etc.)
++++ severity: MEDIUM
+      values.relaxed:
```
Comment (lucadonnoh) [inline packages/backend/discovery/cbridge/ethereum/diffHistory.md:25]: so being relaxed means to have broader permissions, right?

Comment (lucadonnoh) [inline packages/backend/discovery/cbridge/ethereum/diffHistory.md:25]: so being relaxed means to have broader permissions, right?

Comment (sekuba) [reply packages/backend/discovery/cbridge/ethereum/diffHistory.md:25]: the Sentry being relaxed means all Governors behind the Sentry (Sentry-Governors in my description) inherit the full permissions and functions of the Sentry, who itself is a Governor and full pauser in the bridges (but there are also other govs in the bridges that are without sentry)

Comment (sekuba) [reply packages/backend/discovery/cbridge/ethereum/diffHistory.md:25]: Plus there is SimpleGovernance for upgrades, which is unrelated
A diagram would probably be interesting here

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**